src/command_modules/azure-cli-role/azure/cli/command_modules/role/custom.py

```diff
@@ -52,7 +52,7 @@ def list_role_definitions(name=None, resource_group_name=None, resource_id=None,
 
 helps['role create'] = """
             type: command
-            short-summary: Create a custom role 
+            long-summary: Create a custom role 
```

Comment:
>ong-summary: Create a custom role  [](start = 13, length = 34)

I think you were right to have this in the short-summary. Every command should have a short summary so that the group help is populated. Have a big blank is conspicuous even if the name of the command is self-descriptive. After all, think of all our awesome name arguments. :) 

The way the docstring is parsed into help is that the first sentence is the short summary and anything else is the long summary, so if your help was parsed from a docstring, "Create a custom role." would in fact be the short summary.